terraform/env/sit.env

```diff
@@ -1,4 +1,30 @@
 export AWS_REGION="us-west-2"
 export BACKEND_BUCKET="podaac-services-sit-terraform"
 
+export TF_VAR_confluence_app_version="2.0.0"
```

Comment:
Are we sure we want to set this app version based on deployment? Should this reflect the version of the "core deployment"?